src/app/app.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { TestBed, async } from '@angular/core/testing';
```

Comment:
We could omit the whole file. No need to test the playground :smiley:

Reply:
also true :)